starter-code/src/components/Row.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Button } from './Button';
+
+export class Row extends React.Component{
+    delete(ctx){
+        ctx[0].deleteItem(ctx[1])
+    }
+    render(){
+        return(
```

Comment:
yo incluiría una destructuración de objeto tal que así `const {img, name, popularity, ctx} = this.props` así luego en la variables te ahorras el escribir el `this.props`

```
render(){
        const {img, name, popularity, ctx} = this.props
        return(
            <tr>
                <th><img width="100px" src={img} alt={name}/></th>
                <th>{name}</th>
                <th>{popularity}</th>
                <th><Button func={()=>this.delete(ctx)}>Delete</Button></th>
            </tr>
        )
    }
```